x-pack/plugins/aiops/public/components/change_point_detection/charts_grid.tsx

```diff
@@ -105,7 +109,10 @@ export const ChartsGrid: FC<{
                 </EuiFlexItem>
                 <EuiFlexItem grow={false}>
                   <EuiText color={'subdued'} size={'s'}>
-                    {v.fn}({v.metricField})
+                    {v.fn}
+                    {i18n.translate('xpack.aiops.chartsGrid.TextLabel', { defaultMessage: '(' })}
+                    {v.metricField}
+                    {i18n.translate('xpack.aiops.chartsGrid.TextLabel', { defaultMessage: ')' })}
```

Comment:
Do we need to translate the parenthesis? If yes maybe this could be changed into a single translated message with `fn` and `metricField` being passed in as values. At the moment both messages have the same id that's why we see the CI error.

Reply:
No, it's a bug in the eslint I discovered yesterday. It was wrapping `(` and `)` with i18n. Pushed the [fix](https://github.com/elastic/kibana/pull/169962/commits/7b32706370c2ad32878cefeb35904f125c5c6b15)